src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java

```diff
@@ -205,21 +223,22 @@ private void controlDrivetrain() {
                     desiredHeading = getFieldToSource.get();
                     break;
                 case NONE:
+                    break;
```

Comment:
You don't actually need to add this because it will fall through into default by design. However, you don't need to remove it now that it's added (more explicit code is good). Feel free to resolve this comment once you read it